apps/wallet-mobile/src/features/Transactions/TxHistoryNavigator.tsx

```diff
@@ -160,7 +160,7 @@ export const TxHistoryNavigator = () => {
                     }}
                   />
 
-                  <Stack.Screen name="history-details" options={{title: ''}}>
+                  <Stack.Screen name="tx-details" options={{title: ''}}>
```

Comment:
I think title is missing

Reply:
hence ur touching it, please check with Product the tx-details title, right now it is the tx-date, I remember they wanted to change it to Transaction Details.

Reply:
It was already like that because it's then modified via a hook inside the component. But I'm removing the options.